src/Snippets/Docs/CircuitBreaker.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Net;
+using System.Net.Http;
+using Polly;
+using Polly.CircuitBreaker;
+using Snippets.Docs.Utils;
+
+namespace Snippets.Docs;
+
+#pragma warning disable S103 // Lines should not be too long
+
+internal static class CircuitBreaker
+{
+    public static async Task Usage()
+    {
+        #region circuit-breaker
+
+        // Use the default Circuit Breaker options with pre-configured settings:
+        //
+        // FailureRatio: 0.1 — Opens the circuit if 10% of the calls fail
+        // BreakDuration: 30 seconds — Duration the circuit stays open before another try
+        // MinimumThroughput: 100 — Minimum number of calls required before the circuit can break
```

Comment:
```suggestion
        // MinimumThroughput: 100 — Minimum number of calls required before the circuit can break/opens
```

Reply:
Will drop these details and add link to default options in follow-up. (to reduce duplications)